pkg/ingester/instance_limits.go

```diff
@@ -13,6 +13,8 @@ var (
 	errMaxSeriesLimitReached          = errors.New("cannot add series: ingesters's max series limit reached")
 	errTooManyInflightPushRequests    = errors.New("cannot push: too many inflight push requests in ingester")
 	errTooManyInflightQueryRequests   = errors.New("cannot push: too many inflight query requests in ingester")
+
+	pushErrTooManyInflightRequests = "errTooManyInflightRequests"
```

Comment:
Thanks. I think we can omit 'err' from the reason label, since the metric name itself contains `error`.
```suggestion
	pushErrTooManyInflightRequests = "tooManyInflightRequests"
```